lib/cucumber/runtime.rb

```diff
@@ -36,6 +36,7 @@ class FileNotFoundException < FileException
 
   class FeatureFolderNotFoundException < RuntimeError
     def initialize(path)
+      super
```

Comment:
Could this affect the path assign (Usually safe supers go at the bottom)

Reply:
I'll check